go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/hashicorp/packer-plugin-proxmox
 go 1.17
 
 require (
-	github.com/Telmate/proxmox-api-go v0.0.0-20210402150908-b9cc6607c070
+	github.com/Telmate/proxmox-api-go v0.0.0-20210825163308-5e4c0d698a78
```

Comment:
Bumping dependency to get access to DeleteVolume to cleanup generate ISO files. This also fixes virtual machines not correctly being deleted (noticed on my Proxmox instance v7.0-9)